### PR TITLE
model(llm): fix refactor architecture attention ops

### PR DESCRIPTION
**Pull request by rebel-kblee**
# Pull Request Description

remaining issue for colqwen -> custom class
- add _use_learned_pos_emb flag for wrapper

## Type of Change

- [ ] New Model Support
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Performance improvement
- [ ] Code refactoring
- [ ] Other (please describe):

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have added tests that prove my fix is effective or that my feature works (If needed)

## Related Issues

----
# Conventional commit
```
type(optional scope): description
```
----
# Type candidate
  - Model Updates
    - `model`: Adding New models or Bugfix for existing models
      - ex) Add LlavaNext 
      - ex) Bugfix Whisper
  - Enhancements
    - `performance`: Optimizing some models or this library itself
      - ex) Loading RBLNModel faster
      - ex) Optimizing Memory Usage of DecoderOnlyModel
  - Code Refactor
    - `refactor`: Re-arrange class architecture, or more.
      - ex) Refactor Seq2Seq
  - Documentation
    - `doc`: Update docstring only     
  - Library Dependencies
    - `dependency`: Update requirements, something like that.
  - Other
    - `other`: None of above.
      - ex) ci update
      - ex) pdm update

### CODE REVIEW

File: src/optimum/rbln/transformers/models/decoderonly/decoderonly_architecture.py [hunk 1]
```diff
@@ -231,14 +231,15 @@ def get_rbln_causal_lm_class(self):
     def convert_to_rbln_causal_lm(self, causal_lm: PreTrainedModel, max_seq_len: int):
         new_layers = []
         for layer_idx, layer in enumerate(self.get_decoder_layers(causal_lm)):
+            is_sliding = layer_idx in self.sliding_window_layers
             new_self_attn = self.get_rbln_attn_class()(
                 self.get_attn_layer(layer),
-                self.use_attention_mask,
+                self.use_attention_mask if is_sliding else False,
```
Comment (rebel-jongho) [inline src/optimum/rbln/transformers/models/decoderonly/decoderonly_architecture.py:237]: is it right? @rebel-thkim 

if the layer is .. 
- sliding window -> attn mask == True
- normal attn -> attn mask == self.use_attn_mask

**Review by rebel-thkim — APPROVED** [review]
👍
